src/lib.rs

```diff
@@ -168,6 +167,9 @@ use certificate::X509Certificate;
 use error::X509Result;
 use revocation_list::CertificateRevocationList;
 
+/// Default maximum object size (2^32)
+pub const MAX_OBJECT_SIZE: usize = 4_294_967_295;
```

Comment:
nit: Isn't this 2^32-1?

2^32 is 4_294_967_296

Reply:
Indeed
Note: this constant is not used anymore (`asn1-rs` takes care of reading objects), so I'm considering removing it.